### PR TITLE
Homepage redesign v1

### PR DESCRIPTION
**Pull request by benpixel**
@shime check it out!

### CODE REVIEW

File: server/views/index.handlebars [hunk 1]
```diff
@@ -1,81 +1,123 @@
...
+    // var tick = function() {
+    //   x.style.opacity = +x.style.opacity + (new Date() - last) / 400;
+    //   last = +new Date();
+    //
+    //   if (+x.style.opacity < 1) {
+    //     (window.requestAnimationFrame && requestAnimationFrame(tick)) || setTimeout(tick, 16);
+    //   }
+    // };
+
+    tick();
```
Comment (shime) [inline server/views/index.handlebars:116]: should this be commented out too?

**Comment by benpixel** [comment]
Feel free to delete whats commented out. Removing class is for animating "thunder" entrance - whole thing could be moved into proper file and have a tiny timeout like 0.1-0.2s. Feel free to adjust how you think is best! 

---- On Sun, 23 Jul 2017 22:31:25 +0200 notifications@github.com wrote ----

@shime commented on this pull request.

In server/views/index.handlebars:

> +    console.log(x)
+
+    x.classList.remove("closedCircuit");
+
+
+    // var last = +new Date();
+    // var tick = function() {
+    //   x.style.opacity = +x.style.opacity + (new Date() - last) / 400;
+    //   last = +new Date();
+    //
+    //   if (+x.style.opacity < 1) {
+    //     (window.requestAnimationFrame && requestAnimationFrame(tick)) || setTimeout(tick, 16);
+    //   }
+    // };
+
+    tick();
should this be commented out too?

**Comment by dinodsaurus** [comment]
Awesome!

**Comment by shime** [comment]
Thanks for this, Benjamin! I owe you so much beer. 🍻

**Comment by benpixel** [comment]
@shime @dinodsaurus Would this work? I'm not sure where to put it and I'm 100% sure it's not by the ES9999 standards. :D

  ```javascript
(function() {
    setTimeout(function(){ document.getElementById("highVoltage").classList.remove("closedCircuit"); }, 200);
  })();
```

**Comment by shime** [comment]
@benjam1n yeah, that would also work

**Pull request merged**